### DISCUSSION TITLE
Permission Denied

### DISCUSSION THREAD

**Discussion by rampart51** [Security-Onion-Solutions/securityonion#2661] [Unsupported Versions] [answered]
I am running v2.3.2 Standalone.  I can cd to /opt/so but if I try to go one more level to 'log' (cd /opt/so/log or /opt/so/saltstack) I get a 'permission denied'.  I am in the system with the root account established during the setup.  Any ideas as to what the issue could be?  I see that the particular directory is owned by socore but am not familiar with that account.

**Comment by TOoSmOotH** [comment, marked as answer]
you need to be root to go in there. sudo su -

**Reply by rampart51** [reply]
of course it was that simple... thank you
